docs/customizations/helm.rst

```diff
@@ -421,606 +433,15 @@ NIC Configuration Operator Helm chart customization options can be found `here <
-     - string
-     - `"15b3"`
-     -
-   * - sriovDevicePlugin.useCdi
-     - bool
-     - `false`
-     - Enable Container Device Interface (CDI) mode. **NOTE**: NVIDIA Network Operator does not configure container runtime to enable CD.
-   * - sriovDevicePlugin.version
-     - string
-     - `"v3.7.0"`
-     - SR-IOV Network device plugin version
-
-===================================================
-SR-IOV Network Device Plugin Resource Configuration
-===================================================
-
-Consists of a list of RDMA resources, each with a name and a selector of RDMA capable network devices to be associated with the resource. Refer to `SR-IOV Network Device Plugin Selectors <https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin#device-selectors>`_ for supported selectors.
-
-.. code-block:: yaml
-
-    resources:
-    - name: hostdev
-      vendors: [15b3]
-    - name: ethernet_rdma
-      vendors: [15b3]
-      linkTypes: [ether]
-    - name: sriov_rdma
-      vendors: [15b3]
-      devices: [1018]
-      drivers: [mlx5_ib]
-
-=============
-IB Kubernetes
-=============
-
-ib-kubernetes provides a daemon that works in conjunction with the `SR-IOV Network Device Plugin <https://github.com/openshift/sriov-network-operator>`_. It acts on Kubernetes pod object changes (Create/Update/Delete), reading the pod's network annotation, fetching its corresponding network CRD and reading the PKey. This is done in order to add the newly generated GUID or the predefined GUID in the GUID field of the CRD cni-args to that PKey for pods with ``mellanox.infiniband.app`` annotation.
-
-.. list-table::
-   :header-rows: 1
-
-   * - Name
-     - Type
-     - Default
-     - Description
-   * - ibKubernetes.deploy
-     - bool
-     - `false`
-     - Deploy IB Kubernetes.
-   * - ibKubernetes.image
-     - string
-     - `"ib-kubernetes"`
-     - IB Kubernetes image name.
-   * - ibKubernetes.pKeyGUIDPoolRangeEnd
-     - string
-     - `"02:FF:FF:FF:FF:FF:FF:FF"`
-     - Maximal available GUID value to be allocated for the pod.
-   * - ibKubernetes.pKeyGUIDPoolRangeStart
-     - string
-     - `"02:00:00:00:00:00:00:00"`
-     - Minimal available GUID value to be allocated for the pod.
-   * - ibKubernetes.periodicUpdateSeconds
-     - int
-     - `5`
-     - Interval of periodic update in seconds.
-   * - ibKubernetes.repository
-     - string
-     - `"ghcr.io/mellanox"`
-     - IB Kubernetes image repository.
-   * - ibKubernetes.ufmSecret
-     - string
-     - `""`
-     - Name of the Secret with the NVIDIA UFM access credentials, deployed in advance.
-   * - ibKubernetes.version
-     - string
-     - `"v1.1.0"`
-     - IB Kubernetes version.
-
-==========
-UFM Secret
-==========
-
-IB Kubernetes must access `NVIDIA UFM <https://www.nvidia.com/en-us/networking/infiniband/ufm/>`_ in order to manage pods' GUIDs. To provide its credentials, the secret of the following format should be deployed in advance:
```

Comment:
Make sure we have similar explanation/link

Reply:
Added in godoc